compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__unknown_capital_case_tuple_variable.snap

```diff
@@ -0,0 +1,14 @@
+---
+source: compiler-core/src/type_/tests/errors.rs
+expression: "let triple = #(1, 2.2, \"three\") let #(One, _, _) = triple"
+---
+error: Unknown variable
+  ┌─ /src/one/two.gleam:1:39
+  │
+1 │ let triple = #(1, 2.2, "three") let #(One, _, _) = triple
+  │                                       ^^^
+
+The name `One` is not in scope here.
+
+This pattern is matching a constructor named `One`, but there is no
+constructor in scope with that name.
```

Comment:
This error message says the same thing twice. Is that an improvement on what it said before?